app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,27 @@
   </head>
 
   <body>
+    <nav>
+      <div class="container">
+        <div class="row">
+          <div class="col-4 left">
+            <%= link_to "Pets", "/pets/"%> 
+          </div>
+          <div class="col-4 left">
+            <%= link_to "Shelters", "/shelters/"%> 
+          </div>
+          <div class="col-4 left">
+            <p>Favorited Pets: <%= favorite.total_count %></p>
+          </div>
+        </div>
+      </div>
+   </nav>
```

Comment:
This displays our nav bar at the top of every page instead of needing to add the code in each page individually.